fmpz/aors_ui.c

```diff
@@ -124,13 +124,12 @@ _fmpz_add_mpn_1(fmpz_t f, const mp_limb_t * glimbs, mp_size_t gsz, mp_limb_t x)
 
     if (mf->_mp_alloc < (gabssz + 1)) /* Ensure result fits */
     {
-        mf->_mp_d = flint_realloc(mf->_mp_d, sizeof(mp_limb_t) * (gabssz + 1));
-        mf->_mp_alloc = gabssz + 1;
+        mp_limb_t * tmp = flimbs;
+        flimbs = _mpz_realloc(mf, sizeof(mp_limb_t) * (gabssz + 1));
```

Comment:
_mpz_realloc takes its size in limbs?
I don't see any gmp function that takes the size in bytes, for example, mpz_realloc2 takes the size in bits.